.gitignore

```
@@ -1,4 +1,4 @@
-config/*
+/config
```

```suggestion
/config
config/*
```

To keep existing and allow for your use-case we need both right?

---

No?

`/config` <- can be both a file or folder?

---

dark magic